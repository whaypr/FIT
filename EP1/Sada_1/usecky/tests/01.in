1
10.0 10.0 13.0 14.0
3
0.1 0.1 0.2 0.2
1.0 1.0 2.0 2.0
2.0 2.0 3.0 3.0
9
0 0 0 0
0 0 0 0
0 0 0 0
5 5 80 9503
5 5 80 9503
5 5 80 9503
5 5 80 9503
5 5 80 9503
0 0 0 0
0