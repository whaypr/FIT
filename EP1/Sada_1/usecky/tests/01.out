1 usecek ma delku 5.00.
2 usecek ma delku 1.41.
5 usecek ma delku 9498.30.
